src/content/docs/style-guide/article-templates/whats-new-style-guidelines.mdx

```diff
@@ -137,4 +141,11 @@ Ordered list
 2. Step two
 3. Step three
 ```
+## Best practices
+
+Here are some tips to ensure that your posting goes smoothly:
 
+* **Publishing timing:** If you are a writer publishing a post from a contributor, pay attention to when the posts should be made public. You may need to stage the docs in preparation for a future release date.
+* **Links:** Be sure to use absolute links for all your posts, even for links that refer to the docs site.
+* **Filenames:** Don't include any spaces in your filenames. If you are a writer removing filename spaces from a contributor, be sure to ensure the name is unique so you avoid collisions with existing posts. One way to do this is to include the date in the filename.
```

Comment:
"be sure to ensure" seems redundant to me. Maybe just "from a contributor, make sure the name..."?

Reply:
@barbnewrelic, thanks for catching that! I just pushed up a correction!